stats.go

```diff
@@ -115,8 +118,13 @@ func (t *timer) Record(interval time.Duration) {
 // NullStatsReporter is an implementatin of StatsReporter than simply does nothing.
 var NullStatsReporter StatsReporter = nullStatsReporter{}
 
-func (r nullStatsReporter) ReportCounter(name string, tags map[string]string, value int64)          {}
-func (r nullStatsReporter) ReportGauge(name string, tags map[string]string, value int64)            {}
-func (r nullStatsReporter) ReportTimer(name string, tags map[string]string, interval time.Duration) {}
+func (r nullStatsReporter) ReportCounter(name string, tags map[string]string, value int64) {
+}
+func (r nullStatsReporter) ReportGauge(name string, tags map[string]string, value int64) {
+}
+func (r nullStatsReporter) ReportTimer(name string, tags map[string]string, interval time.Duration) {
+}
+func (r nullStatsReporter) Flush() {
```

Comment:
i kind of like the old formatting better where it just had the single line + aligned `{}`